aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -26,6 +26,41 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.bls12381.per_msg_hashing, "bls12381.per_msg_hashing", 1_540_000 * MUL],
     [.bls12381.per_byte_hashing, "bls12381.per_byte_hashing", 50 * MUL],
 
+    [.groups.bls12_381.pairing_product_per_pair, "groups.bls12_381.pairing_product_per_pair", 1_782_800 * MUL],
```

Comment:
Just leaving a comment here for us to resolve later: We gotta make sure these numbers are computed by benchmarking on x86_64 relatively to the SHA2-256 baseline that @vgao1996 gased up for us.